src/main.tsx

```diff
@@ -10,10 +13,27 @@ async function enableMocking() {
   }
 }
 
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Dashboard />,
+  },
+  {
+    path: "/jsonformsplans",
+    element: <JsonFormsPlans />,
+  },
+  {
+    path: "/plans",
+    element: <Plans />,
+  },
+]);
+
 enableMocking().then(() => {
   createRoot(document.getElementById("root")!).render(
     <StrictMode>
-      <App />
+      <ThemeProvider theme={DiamondTheme} defaultMode="light">
```

Comment:
Should we respect the current browser setting and then add the mode toggle button? Not crucial now

Reply:
I've added a colour mode switch to the Navbar. The placement isn't ideal. Again, there are changes upstream going on that need releasing for more control over that. To use the browser's colour scheme, I'll look into implementing it in SciReactUI.